ansible/tasks/initdb.yml

```diff
@@ -2,20 +2,20 @@
 # This task will initialize the immortal DBs in the database account.
 # This step is usually done only once per account.
 
-- name: check if the immortal {{ db_auth }} db with {{ db_provider }} exists?
+- name: check if the immortal {{ db.whisk.auth }} db with {{ db_provider }} exists?
   uri:
-    url: "{{ db_protocol }}://{{ db_host }}:{{ db_port }}/{{ db_auth }}"
+    url: "{{ db_protocol }}://{{ db_host }}:{{ db_port }}/{{ db.whisk.auth }}"
```

Comment:
@markusthoemmes I know. I was referring to this one as an example for quoting. seems like ansible treats task titles different then.